tests/main/snapctl-disable-services-install/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Check that `snapctl stop --disable` actually stops services on install
+
+prepare: |
+    snap pack "$TESTSLIB"/snaps/snapctl-disable-services-install
+
+restore: |
+    rm -f snapctl-disable-services-install_1.0_all.snap
+
+execute: |
+    echo "Verify that the install hook finishes while installing"
+    snap install --dangerous snapctl-disable-services-install_1.0_all.snap
```

Comment:
You can drop `prepare` and `restore` sections and do:
```
# shellcheck source=tests/lib/snaps.sh
. "$TESTSLIB"/snaps.sh
install_local snapctl-disable-services-install
```

Reply:
Oh and the snap is named `snapctl-disable-services-install-hook`

Reply:
Ah that explains @zyga's comment about the missing file... Thanks!